uxkit-screenshare/src/main/java/com/voxeet/uxkit/screenshare/ScreenShareMediaPresentationProvider.java

```diff
@@ -0,0 +1,53 @@
+package com.voxeet.uxkit.screenshare;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.voxeet.uxkit.common.logging.ShortLogger;
+import com.voxeet.uxkit.common.UXKitLogger;
+import com.voxeet.uxkit.presentation.provider.AbstractMediaPlayerProvider;
+
+/**
+ * Manage and create screenshare videos
+ */
+public class ScreenShareMediaPresentationProvider extends AbstractMediaPlayerProvider<ScreenShareMediaPresentationView> {
+
+    private final static ShortLogger Log = UXKitLogger.createLogger(ScreenShareMediaPresentationProvider.class);
```

Comment:
lower case

Reply:
I used Log to reflect the standard singleton. Here it would be uppercase

Reply:
Yea, we still should follow the naming convention. If its variable it should be lower case.